src/test/java/edu/cornell/kfs/kns/datadictionary/validation/fieldlevel/CuEmailAddressValidationPatternTest.java

```diff
@@ -27,20 +27,26 @@ public class CuEmailAddressValidationPatternTest {
 
     @Before
     public void setUp() throws Exception {
-        pattern = PowerMockito.spy(new EmailAddressValidationPattern());
-        String emailRegEx = getProperty(PATTERN_CONSTRAINT);
-        PowerMockito.doReturn(emailRegEx).when(pattern, "getRegexString");
+        final String emailRegEx = getProperty(PATTERN_CONSTRAINT);
+        pattern = new EmailAddressValidationPattern() {
+            @Override
+            public boolean matches(String value) {
+                if (value == null) {
+                    return false;
+                }
+                return Pattern.compile(emailRegEx).matcher(value).matches();
+            }
+        };
     }
     
     private String getProperty(String key) {
         String value = null;
         Properties properties = new Properties();
-        try {
-            InputStream in = Thread.currentThread().getContextClassLoader().getResourceAsStream(CU_APPLICATION_RESOURCES_PATH);
+        try (InputStream in = Thread.currentThread().getContextClassLoader().getResourceAsStream(CU_APPLICATION_RESOURCES_PATH);) {
             properties.load(in);
             value = properties.getProperty(key);
         } catch (IOException e) {
-            
+            e.printStackTrace();
```

Comment:
If you need to log something here, I would suggest using a Log4J logger instead.